Source/Priority.swift

```diff
@@ -50,20 +50,20 @@ public enum Priority: ExpressibleByFloatLiteral, ExpressibleByIntegerLiteral, Eq
         }
     }
 
-    public init(floatLiteral value: LayoutPriority) {
-        self.init(value)
+    public init(floatLiteral value: Float) {
+        self.init(Int(value))
     }
 
     public init(integerLiteral value: Int) {
         self.init(value)
     }
 
     public init(_ value: Int) {
-        self = .custom(LayoutPriority(value))
+        self = .custom(LayoutPriority(Float(value)))
     }
 
     public init<T: BinaryFloatingPoint>(_ value: T) {
-        self = .custom(LayoutPriority(value))
+        self = .custom(LayoutPriority(Float(value)))
```

Comment:
Is the `Float(value)` necessary? Is `LayoutPriority` no longer an alias for `Float`?